docs/user-guide/pod-preset/index.md

```diff
@@ -0,0 +1,540 @@
+---
+assignees:
+- jessfraz
+title: Pod Preset
+---
+
+Objects of type `podpreset` allow for injecting certain information into pods
```

Comment:
"You can use a `podpreset` object to inject certain information into pods at creation time. This information can include secrets, volumes, volume mounts, and environment variables."